### PR TITLE
Create basic front end

### PR DESCRIPTION
**Pull request by aKrisnanto**
Takes data and puts it onto a table.  Currently very inflexible as data format was still unknown.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "mysql": "^2.18.1",
```
Comment (Jaredhw) [inline client/package.json:10]: This is outdated, should be mysql2. "mysql2": "^2.2.5"

**Review by Jaredhw — APPROVED** [review]

**Review by AA666666 — APPROVED** [review]

**Comment by AA666666** [comment]
very informative. Good job

File: client/src/App.js [hunk 2]
```diff
@@ -1,59 +1,82 @@
...
+          console.log("Connected.")
+          //Queries done here
+          con.query("SELECT * FROM MEMBERS", function (err, result) {
+              if (err) throw err;
+              Object.keys(result).forEach(function (key) {
+                  this.queriedResult = result[key];
+              });
+          })
       });
   }
```
Comment (JadenBalogh) [inline client/src/App.js:41]: Database queries happen on the server, not the client.

File: client/src/App.js [hunk 3]
```diff
@@ -1,59 +1,82 @@
...
-    this.state = {
-      value: "",
-      response: ""
-    };
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
+      super(props);
+      this.state = { total: 0 }
+      var rows = props.data;
+      var queriedResult = [];
```
Comment (JadenBalogh) [inline client/src/App.js:11]: queriedResult should be part of the state

File: client/src/App.js [hunk 4]
```diff
@@ -1,59 +1,82 @@
-import React, { Component } from 'react';
-import './App.css';
+import React from 'react';
+import './index.css';
+import reportWebVitals from './reportWebVitals';
+import MemberList from './MemberList';
```
Comment (JadenBalogh) [inline client/src/App.js:4]: In React, we either create a Class or a Function component, not both. Each component should contain all of its state and functionality within itself (and potentially containing child components). Right now you have two MemberList components, with one nested within the other. These should be merged into one component.

**Review by JadenBalogh — APPROVED** [review]
These components will need to be cleaned up and updated for a working product, but are good enough for this first task which is mostly learning-focused. Nice work, honestly!

**Pull request merged**